draft-ietf-rats-msg-wrap.md

```diff
@@ -274,6 +274,13 @@ Since the collection type is recursive, implementations may limit the allowed de
 
 Although initially designed for the composite Attester use case, the CMW collection can be repurposed for other use cases requiring CMW aggregation.
 
+### CMW Collections' role in composite Attester topology
+
+A CMW Collection's tree structure is not required to be a spanning tree of the system's composite Attester topology.
+If a label changes Verifier state beyond a "hint" (e.g., for better Verifier performance or human comprehension), we say that it carries semantic content.
+If a label carries semantic content, the collection itself is a form of evidence that SHOULD be attested.
```

Comment:
Isn't L281 redundant given L282 restates both the content and the requirement level (i.e., SHOULD)?  If so, I suggest we drop it.